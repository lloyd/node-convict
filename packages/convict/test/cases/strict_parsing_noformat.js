'use strict';

exports.conf = {
  badusage: {
    default: 'string'
  }
};

exports.strictParsing = true;
