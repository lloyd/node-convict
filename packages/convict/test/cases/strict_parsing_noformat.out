badusage: format property is missing